### PR TITLE
Edit format consistency

### PR DESCRIPTION
**Pull request by helloiamcait**
Added "." after each step for consistency.

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: migration/mm-google-overview.md [hunk 1]
```diff
@@ -33,11 +33,11 @@ Collaborate all in one place by migrating your Google documents, data, and users
...
+- **Step 2:** [Scan and assess](mm-google-step2-scan-assess.md). Google Drive accounts are scanned automatically for you. Once the scans are complete, download the generated reports and logs to investigate any possible issues that might block your migration.
+- **Step 3:** [Copy to Migrations list](mm-google-step3-copy-to-migrations.md). After a Google Drive has been scanned and determined ready, add them to your migration list.
+- **Step 4:** [Review destination paths](mm-google-step4-review-destinations.md).  We automatically map source paths to any exactly matching destination paths. Ensure content is being copied to the right place by reviewing and modifying as needed for each destination path.
+- **Step 5:** [Map identities](mm-google-step5-map-identities.md).   Map your groups and users in Google Drive to an account in Microsoft 365 to migrate your Google Drive permission settings.
+- **Step 6:** [Migrate and Monitor](mm-google-step6-migrate-monitor.md). After reviewing your migration setup, migrate your Google accounts and monitor the progress
```
Comment (JohanFreelancer9) [inline migration/mm-google-overview.md:40]:
```suggestion
- **Step 6:** [Migrate and Monitor](mm-google-step6-migrate-monitor.md). After reviewing your migration setup, migrate your Google accounts and monitor the progress.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @helloiamcait, could you please commit the additional suggested change? Many thanks.

**Comment by helloiamcait** [comment]
> Apologies @helloiamcait, could you please commit the additional suggested change? Many thanks.

Absolutely! I've committed the suggested change. Thanks!

**Comment by scanum** [comment]
@JoanneHendrickson 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**